BotModules/ScreechBot/index.js

```diff
@@ -0,0 +1,176 @@
+const {
+  LOOPDELAY, // Delays the watch loop on the server by milliseconds to reduce server overhead. larger delay is less responsive.
+  GREETING_DELAY // Give some time before the greeting is played to allow the user to connect to the voice chat successfully.
+} = require('./config.json');
+
+const EventEmitter = require('events');
+const path = require('path');
+
+//Custom event emitter for capturing events from the watch loop
+class VoiceEmitter extends EventEmitter {}
+voiceChat = new VoiceEmitter();
+
+//Simple command register. Allows for reuse of code and ease of adding in new commands
+// - keyword: is the command keyword than spawns the action - TODO: create a way to receive arguments
+// - helptext: is the text that will show up when the user calls the /help command. This should describe what the command does and how the user should use it.
+// - action: is the function that is called when the keyword is typed in chat. Action function is of the form function(msg) with msg being the discord msg that spawned the command.
+var commands = [{
+  keyword: '/help',
+  helptext: 'When you are a noob and need to reference the commands',
+  action: help
```

Comment:
I would like to breakout the help command into its separate submodule so we can use it across all the bots. This might require a separate PR though due to the way things are coded here.